content/messages/index.textile

```diff
@@ -40,5 +40,56 @@ The following are the properties of a message:
 - clientId := The "ID of the client":/auth/identified-clients that published the message.
 - connectionId := The ID of the connection used to publish the message.
 - timestamp := The timestamp of when the message was received by Ably, as milliseconds since the Unix epoch.
-- extras := A JSON object of arbitrary key-value pairs that may contain metadata, and/or ancillary payloads. Valid payloads include those related to "Push Notifications":/push, "deltas":/channels/options/deltas and  and headers.
+- extras := A JSON object of arbitrary key-value pairs that may contain metadata, and/or ancillary payloads. Valid payloads include those related to "Push Notifications":/push, "deltas":/channels/options/deltas and headers.
 - encoding := This is typically empty, as all messages received from Ably are automatically decoded client-side using this value. However, if the message encoding cannot be processed, this attribute contains the remaining transformations not applied to the data payload.
+
+h2(#routing). Message routing and conflation syntax
+
+Ably uses common syntax for to select which messages are routed to integrations and for assessing which messages to apply conflation to. The following properties and features use this syntax:
+
+* @routingKey@ for "AMQP":/general/firehose/amqp-rule and "Kafka":/general/firehose/kafka-rule integrations
+* @partitionKey@ for "AWS Kinesis":/general/firehose/kinesis-rule integrations
+* @conflationKey@ for "message conflation":#conflation
```

Comment:
#conflation link does not work. I tried to find where it may have meant to go, but wasn't able to.

Reply:
There's a note in the description - it'll exist in a separate PR that I'll merge with this one 🙂